local/bin/py/build/pull_rbac.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import sys
+import requests
+import json
+from os import getenv
+
+def pull_rbac():
+  api_endpoint = 'https://app.datadoghq.com/api/v2/permissions'
+  headers = {'DD-API-KEY': sys.argv[1], 'DD-APPLICATION-KEY': sys.argv[2]}
+  formatted_permissions_dict = {}
+
+  r = requests.get(api_endpoint, headers=headers)
+
+  if r.status_code == requests.codes.ok:
+    json_result = r.json()
+    data = json_result['data']
```

Comment:
Just an fyi with dicts its pretty easy to access with square brackets and its easy to do when switching languages but there might be situations where you provide a fallback should something be wrong. e.g  `json_result.get('data', [])`

Reply:
Done, thanks!